app/components/chart/settings.js

```diff
@@ -118,7 +118,7 @@ var Settings = React.createClass({
   },
   isMissingSettings: function() {
     var data = this.props.patientData;
-    if (_.isEmpty(data.grouped.settings)) {
+    if (_.isEmpty(data) || _.isEmpty(data.grouped) || _.isEmpty(data.grouped.settings)) {
```

Comment:
As I commented in your fork, maybe using the `utils.getIn` function would be nice for doing this check. If you disagree, LMK. My feelings are not super strong on this one.